packages/experiments/src/components/Chiclet/Chiclet.test.tsx

```diff
@@ -62,3 +63,27 @@ describe('Chiclet', () => {
     expect(tree).toMatchSnapshot();
   });
 });
+
+describe('Class Change', () => {
```

Comment:
Woo thanks for writing a test! I just have one proposition - can we have a test to more specifically check the functionality of the `preview` prop on the Chiclet? The test could render a Chiclet with a `preview`, and then ensure that that preview element gets the `maxWidth: 100%` style from the `className`. Let me know if you have questions about this.